contracts/interfaces/IOrderMixin.sol

```diff
@@ -133,12 +134,13 @@ interface IOrderMixin {
     ) external payable returns(uint256 makingAmount, uint256 takingAmount, bytes32 orderHash);
 
     /**
-     * @notice Same as `fillOrder` but allows to specify extensions that are used for the order
+     * @notice Same as `fillOrder` but allows to specify arguments that are used by the taker.
      * @param order Order quote to fill
      * @param r R component of signature
      * @param vs VS component of signature
      * @param amount Taker amount to fill
-     * @param takerTraits Specifies threshold as maximum allowed takingAmount when takingAmount is zero, otherwise specifies minimum allowed makingAmount. Top-most bit specifies whether taker wants to skip maker's permit.
+     * @param takerTraits Specifies threshold as maximum allowed takingAmount when takingAmount is zero, otherwise specifies
+     * minimum allowed makingAmount. The 2nd (0 based index) highest bit specifies whether taker wants to skip maker's permit.
```

Comment:
It seems that docgen may generate markdown text containing "\n" and "*". We need to verify and ensure that this is not the case.